contrib/renew-client-cert

```diff
@@ -39,7 +39,20 @@ rotate() {
 	if [ -n "$cert_ids" ]; then
 		extra_args="$extra_args --pkcs11-cert-ids=$cert_ids"
 	fi
-	exec $fioconfig -c $SOTA_DIR renew-cert $extra_args $est_server $rotation_id 
+	# fioconfig's http logic is already doing back off logic with intervals:
+	#  1, 2, 5, 13, 30 (seconds)
+	# We should sleep in *minutes* given
+	for x in 0 1 2 5 13 ; do
```

Comment:
I wouldn't go too high (13) here as this will block the main loop as well, right? 5 after 1 and 2 should probably be enough.

Also, do we indeed want to sleep 0?

Reply:
Indeed, 0 is needed as the first call was removed above this one, so all good.